std_msgs/Header header
uint32 num_features
FeatureData[] features
